package.json

```diff
@@ -14,9 +14,9 @@
     "asar": "^0.13.0",
     "bower": "^1.3.12",
     "electron": "^1.6.1",
-    "electron-builder": "^17.0.1",
+    "electron-builder": "^18.8.1",
```

Comment:
Interesting. I notice that 18.6.2 is the 'latest public release' according to the main repository. What was your reasoning for choosing this prerelease build vs. the 19.x prerelease builds?